solarwinds_apm/tracer_provider.py

```diff
@@ -0,0 +1,18 @@
+# © 2025 SolarWinds Worldwide, LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at:http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+from opentelemetry.sdk.trace import TracerProvider
+from typing_extensions import override
+
+from solarwinds_apm.oboe.http_sampler import HttpSampler
```

Comment:
While we're not yet setting HttpSampler by default, this import will fail after release unless we also update packaging. Please update [pyproject.toml packages](https://github.com/solarwinds/apm-python/blob/main/pyproject.toml#L76-L85) to include new `"solarwinds_apm.oboe",`

Reply:
Sure. Done